.github/workflows/biome-check.yaml

```
@@ -93,16 +97,15 @@ jobs:
         run: |
-          total_issues=${{ steps.biome-check.outputs.total_issues }}
-          
-          if [ "$total_issues" -eq 0 ]; then
+          final_issues_count=${{ steps.biome-check.outputs.final_issues_count }}
+
+          if [ "$final_issues_count" -eq 0 ]; then
             progress_percentage=100
           else
             progress_percentage=0
           fi
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)